src/routes/AgreementViewRoute/AgreementViewRoute.js

```diff
@@ -206,6 +206,7 @@ class AgreementViewRoute extends React.Component {
 
   static contextType = CalloutContext;
 
+  /* istanbul ignore next */
   componentDidUpdate(prevProps) {
```

Comment:
You can use the `rerender` helper from RTL to trigger the componentDidUpdate. Lets avoid using `istanbul ignore next` here

example usage: https://github.com/folio-org/ui-agreements/blob/master/src/routes/AgreementsRoute/AgreementsRoute.test.js#L226